css/components/alert.css

```diff
@@ -0,0 +1,60 @@
+/*
+|| Alert CSS
+*/
+
+.alert{
+    padding: 1rem 1.5rem;
+    margin:0.2rem;
+    border-radius: 0.5rem;
+    max-width: 100%;
```

Comment:
by default it will take 100% of the width since `div` and `p` are block level elements